### PR TITLE
Use the workspace feature.

### PR DESCRIPTION
**Pull request by SirVer**
Please rm -rf */target */Cargo.lock.

This might speed up CI as well, since now the correct directory is
cached.

### CODE REVIEW

**Comment by SirVer** [comment]
Reminder to myself: Update README (for new target positions), fix `cargo --all test` to `cargo test -all` and change "View billions of points in your browser."

File: ci/script.sh [hunk 1]
```diff
@@ -1,32 +1,15 @@
...
-    cargo build --verbose
-    cargo test
-    popd
-
-    # sdl viewer
-    pushd sdl_viewer
-    cargo build --verbose
-    cargo test
-    popd
+    # root
```
Comment (thomasschiwietz) [inline ci/script.sh]: this is not the root directory but web_viewer. add another popd to get to root

Comment (SirVer) [reply ci/script.sh]: done!

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**